lib/cocoapods/command/install.rb

```diff
@@ -31,12 +31,14 @@ def self.options
         [
           ['--repo-update', 'Force running `pod repo update` before install'],
           ['--deployment', 'Disallow any changes to the Podfile or the Podfile.lock during installation'],
+          ['--clean-install', 'Ignore the contents of the project cache and force a full pod installation'],
```

Comment:
You can also specify "This only applies to projects that use incremental pod installation"

Reply:
True. Good idea